ecommerce/extensions/payment/views/payu.py

```diff
@@ -0,0 +1,241 @@
+# -*- coding: utf-8 -*-
```

Comment:
I assume we didn't apply changes on this file (which have not nice things) but we can review later